atd-etl/cris_import/cris_import.py

```diff
@@ -242,6 +316,59 @@ def download_extract_archives():
         return zip_tmpdir
 
 
+def specify_extract_location():
```

Comment:
I think that there is some duplication here with code just before this line. `specify_extract_location` is defined twice, and I think that there is overlap in `download_extract_archives` and `download_archives`. Might be another duplicate?

Reply:
Thank you Mike for catching this. These dups are the result of me keeping a slightly less refined version of this "pull zips from local directory" mechanism that I had floating in my checkout -- and getting some of those names jumbled as I brought it into a real branch. The dups are removed in 92efb2cc74dffe9c262e8f51e51937c59b50f864.